e2e/fixtures/fdb_cluster_specs.go

```diff
@@ -193,23 +193,23 @@ func (factory *Factory) createPodTemplate(
 					},
 					Env: []corev1.EnvVar{
 						{
-							Name:  "FDB_TLS_CERTIFICATE_FILE",
+							Name:  fdbv1beta2.EnvNameTLSCert,
```

Comment:
I would prefer to define those as constants in the e2e test suite (assuming we only use them in the e2e test suite).

Reply:
that one is used in `NewFdbPodClient` as well, but I will move any which are only used in e2e tests